RNUnifiedContacts/RNUnifiedContacts.swift

```diff
@@ -167,6 +167,45 @@ class RNUnifiedContacts: NSObject {
 
   }
   
+  @objc func deleteContact(identifier: String, callback: (NSObject) -> () ) -> Void {
+    
+    let contactStore = CNContactStore()
+    
+    do {
+      
+      let cNContact = try contactStore.unifiedContactWithIdentifier( identifier, keysToFetch: keysToFetch )
```

Comment:
Can you use the new `getContact` method to get the Contact? That should clean up the double `do / catch` blocks.